src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -42,6 +43,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <!-- TODO: Remove this reference once packages are updated -->
```

Comment:
> I don't see a project.json within the FunctionalTests which adds to the confusion.

https://github.com/dotnet/corefx/pull/12345 removed all of the test project.jsons.  Every test project now implicitly references all of the libraries via a singular project.json.